code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm

```diff
@@ -162,6 +162,12 @@
 		list("Falling Falcons Shoulder Patch", round(scale * 15), /obj/item/clothing/accessory/patch/falcon, VENDOR_ITEM_REGULAR),
 		list("USCM Shoulder Patch", round(scale * 15), /obj/item/clothing/accessory/patch, VENDOR_ITEM_REGULAR),
 		list("Bedroll", round(scale * 20), /obj/item/roller/bedroll, VENDOR_ITEM_REGULAR),
+
+		list("OPTICS", -1, null, null, null),
+		list("Night Vision Optic (Standard Issue Variant)", round(scale * 0), /obj/item/device/helmet_visor/night_vision, VENDOR_ITEM_REGULAR),
```

Comment:
```suggestion
		list("Night Vision Optic (Standard Issue Variant)", 0, /obj/item/device/helmet_visor/night_vision, VENDOR_ITEM_REGULAR),
```

Reply:
Generally against adding this in general. I get what you're going for, but it's entirely unintuitive.

Reply:
Its a pure fluff change, I can remove it if desired, the idea is a small emphasis of how under-equipped the battalion is. 

It may also be possible for staff to increase the vendors NVG count for events or rewards as an IC resource.